src/installation/guides/raspberry-pi.md

```diff
@@ -0,0 +1,201 @@
+# Raspberry Pi
+
+As of 2017-11-30, Void Linux provides prebuilt Raspberry Pi images and rootfs
+tarballs. Compatible images and tarballs exist for RPi 1, RPi 2 and RPi 3, in
+both glibc and musl variants.
+
+You can either:
+
+- write a prebuilt image directly to your SD card (e.g. `dd if=<void-rpi.img>
+   of=/dev/sdX bs=4M ; sync`), but that might give you a limited partition
+   layout that has to be expanded manually afterwards;
+
+or:
+
+- prepare the partition layout of your SD card manually and extract a tarball to
+   your SD card, as described by this guide.
+
+After downloading the chosen file, make sure to check that the sha256 checksum
+is correct. It is available in the `sha256sums.txt` file.
+
+The `rpi-kernel` packages for all Raspberry Pi variants are built from the
+Raspberry Pi Foundation's kernel tree, which should enable you to use everything
+special like Wi-Fi and Bluetooth.
+
+## Raspberry Pi 3: 32-bit or 64-bit?
+
+It is possible (and to some extent advised) to run the RPi 2 images on your RPi
+3, as the RPi 3's CPU supports both Armv8 and Armv7 instructions.
+
+With the RPi 2 image you obviously just get a 32-bit Armv7 system. That is
+totally fine in general, and is what the popular Raspbian does too.
+
+Be aware that some packages are not available for Armv7 and others are not
+available for AArch64, so that might be a guide for your decision which one to
+pick.
+
+## rootfs-install
+
+Grab the latest platformfs tarball (containing the rootfs as well as
+architecture-specific files) for your device from
+<https://alpha.de.repo.voidlinux.org/live/current/>, e.g.
+`void-rpi2-musl-PLATFORMFS-<date>.tar.xz`.
+
+### Preparing the SD card
+
+The SD card must have at least two partitions, one as `FAT` (with partition type
+`0c`) for `/boot` and another one as `ext4`/`f2fs` for `/`.
+
+Prepare the partitions and mount them in a directory:
+
+```
+$ parted /dev/mmcblk0 # change this to match your SD card
+```
+
+Create the FAT partition of 256MB and make it bootable:
+
+```
+$ parted mktable msdos
+$ parted mkpart primary fat32 2048s 256MB
+$ parted toggle 1 boot
+```
+
+Create the rootfs partition, extending to the end of the device:
+
+```
+$ parted mkpart primary ext4 256MB -1
+$ parted quit
+```
```

Comment:
This shouldn't be using parted, and `/` can be pretty much any mountable filesystem, afaik.